OWNERS_ALIASES

```diff
@@ -0,0 +1,12 @@
+# See the OWNERS docs at https://go.k8s.io/owners#owners_aliases
+
+aliases:
+  # Always allow the core ACK maintainers to have access to your repository
+  core-ack-team:
+    - jaypipes
+    - mhausenblas
+    - a-hilaly
+    - RedbackThomson
+    - vijtrip2
+  # TODO: Add your team members to your team controller alias
+  service-team: []
```

Comment:
can you add the aliases from here?
https://github.com/aws-controllers-k8s/sagemaker-controller/blob/main/CODEOWNERS